app-frontend/src/app/components/sceneItem/sceneItem.controller.js

```diff
@@ -1,10 +1,16 @@
 export default class SceneItemController {
-    constructor() {
+    constructor($scope) {
         'ngInject';
+        $scope.$watch(
+            () => this.selected({scene: this.scene}),
+            (selected) => {
```

Comment:
they can, but I feel like that falls into the same category of not doing things like
 ```
if (this)
  do that;
```  
vs
```
if (this) {
  do that;
}